js/main.js

```diff
@@ -687,6 +686,28 @@ YUI.add(
 					}
 				},
 
+				_scopeCommandLog: function(node) {
```

Comment:
I would change the name of the `node` variable to `nodes` just to make it clear it's a nodelist and not a single node element